game.cpp

```diff
@@ -16,39 +16,39 @@ void checkInput(std::string userInput)
 		} 
 	} 
 
-	string command = userInput.substr(0, 2); // get first 2 characters (should be ad or rm)
+	string command = userInput.substr(0, 1); // get first character (should be a or r)
 	string coordinates = userInput.substr(userInput.length() - 2); // get last two characters
 	char xCoordinate = coordinates[0];
 	char yCoordinate = coordinates[1];
 
-	if (isdigit(xCoordinate) && isdigit(yCoordinate) && userInput.size() == 4) { //checks that the last two characters are numbers and that userInput has no other non-space characters  
+	if (isdigit(xCoordinate) && isdigit(yCoordinate) && userInput.size() == 3) { //checks that the last two characters are numbers and that userInput has no other non-space characters  
```

Comment:
this if statement is redundant; isdigit will be checked in line 28, size of userInput is not our concern

Reply:
the size is to make sure that there are no other characters provided in the input besides a/r and the coordinates

Reply:
How about checking the size of userInput at the start of the function